setup/install/providers/aws/aws-ec2.md

```diff
@@ -93,6 +95,25 @@ Option number 1 is useful for creation of user with AWS Access Key and secret. T
 2. Using AWS IAM Roles
 Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations.
 
+### Spinaker AssumeRole Policy
+1. Incase, if you miss to add append the Line 158, then Manually create a Policy by Navigating to Console > IAM > Policies
```

Comment:
Again, hardcoded line number might go bad.

Reply:
Also, I'm not sure what this sentence is saying. I get the "...manually create a policy part" but I don't quite get the beginning.

Reply:
This line is more about, in case if the user miss to update the downloaded template, we are suggesting them to create the policy..